app/views/return-requirements/frequency-reported.njk

```diff
@@ -15,12 +17,48 @@
 {% endblock %}
 
 {% block content %}
+{# Error summary #}
+  {% if error %}
+    {{ govukErrorSummary({
+      titleText: "There is a problem",
+      errorList: [
+        {
+          text: "Select how often readings or volumes are reported",
```

Comment:
Could you reuse your error constant that is set, since this is the same line of text?